### ISSUE TITLE
Does LINC implement OF 1.3 Meters??

### ISSUE THREAD

**Issue by emafermon** [FlowForwarding/of_protocol#86]
I want to know if LINC implements OpenFlow 1.3 Meters.

Thanks

**Comment by mentels** [comment]
@emafermon yes, LINC implements OF1.3 Meters. You can find implementation [here](https://github.com/FlowForwarding/LINC-Switch/blob/master/apps/linc_us4/src/linc_us4_meter.erl).

**Comment by emafermon** [comment]
Ok, thank you
